main/src/com/google/refine/expr/functions/Cross.java

```diff
@@ -71,22 +84,27 @@ public Object call(Properties bindings, Object[] args) {
 
                 if (v instanceof WrappedCell) {
                     return lookup.getRows(((WrappedCell) v).cell.value);
+                } else if (v instanceof Number && targetColumnName.equals(INDEX_COLUMN_NAME)){
+                    return lookup.getRows(((Number) v).intValue());
                 } else {
                     return lookup.getRows(v);
                 }
             }
         }
-        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + " expects a cell or cell value, a project name to look up, and a column name in that project");
+        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + " expects a cell or cell value, a project name to look up (optional), and a column name in that project (optional)");
     }
 
     @Override
     public String getDescription() {
-        return "Looks up the given value in the target column of the target project, returns an array of matched rows, cell will be interpreted as cell.value";
+        return "Looks up the given value in the target column of the target project, returns an array of matched rows. " +
+                "The first argument will be interpreted as cell.value if set to cell. " +
+                "The second argument will be interpreted as the current project name if omitted or set to ''. " +
```

Comment:
@afkbrb Looks like we need to clean up the help text just a bit?  missing the single quote chars `''` on the end of those 2 sentences?  Looks like some kind of rendering issue?

![image](https://user-images.githubusercontent.com/986438/79639659-adb76480-8152-11ea-8ad9-4a29a582dd37.png)

Reply:
What if we flipped this around and instead just got smarter with inspecting the argument values themselves?

GIVEN Only 1 argument
WHEN argument represents a valid column name in current project
THEN  perform the cross with against valid column name in current project

Instead of this unneeded verbosity...

![image](https://user-images.githubusercontent.com/986438/79640080-40f19980-8155-11ea-80d1-78008cbe3bd5.png)

We make it easier with simpler syntax because we know a few things already from argument inspection (where `cell` and `project` are not specified)

![image](https://user-images.githubusercontent.com/986438/79639960-8366a680-8154-11ea-9a98-19302792db89.png)